packages/manager/apps/pci-private-network/src/pages/new/New.page.tsx

```diff
@@ -83,7 +83,7 @@ export default function NewPage(): JSX.Element {
                 __html: translate(
                   'pci_projects_project_network_private_create_error',
                   {
-                    message: e?.message,
+                    message: e?.response?.data?.message || e?.message,
```

Comment:
Is there a specific reason to use `dangerouslySetInnerHTML` ?

vs something along the line:
```
addError(
  t('pci_projects_project_network_private_create_error', {
     message: e?.response?.data?.message || e?.message,
  }),
  true,
);
```
( same for `addSuccess` )

Reply:
because "pci_projects_project_network_private_create_error" translation contains html tags